src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForManagedIdentityParameterBuilder.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Executors;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.TelemetryCore.Internal.Events;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Builder for AcquireTokenForManagedIdentity (used to get token for managed identities).
+    /// See https://aka.ms/msal-net-managed-identity
+    /// </summary>
+#if !SUPPORTS_CONFIDENTIAL_CLIENT
+    [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]  // hide managed identity flow on mobile
+#endif
+    public sealed class AcquireTokenForManagedIdentityParameterBuilder :
+        AbstractManagedIdentityParameterBuilder<AcquireTokenForManagedIdentityParameterBuilder>
+    {
+        private AcquireTokenForManagedIdentityParameters Parameters { get; } = new AcquireTokenForManagedIdentityParameters();
+
+        /// <inheritdoc />
+        internal AcquireTokenForManagedIdentityParameterBuilder(IManagedIdentityApplicationExecutor managedIdentityApplicationExecutor)
+            : base(managedIdentityApplicationExecutor)
+        {
+        }
+
+        internal static AcquireTokenForManagedIdentityParameterBuilder Create(
+            IManagedIdentityApplicationExecutor managedIdentityApplicationExecutor,
+            string resource)
+        {
+            return new AcquireTokenForManagedIdentityParameterBuilder(managedIdentityApplicationExecutor).WithResource(resource);
+        }
+
+        /// <summary>
+        /// Specifies the resource to acquire token for managed identity. 
```

Comment:
Seems like this should be reworded but not exactly sure how. Something like

`Specifies the resource to acquire token for using managed identity`?

or 

`Specifies the resource to use when acquiring a token for the managed identity`?